src/templates/haml/user/balances.html.haml

```diff
@@ -0,0 +1,22 @@
+%div#balances-content
+  %h1= l('Balances')
+  %p
+  - if (@$client_balances > 0) {
+    %table
+      %tr
+        %th= l('Login ID')
+        %th= l('Currency')
+        %th= l('Balance')
+      - for my $balance (@{$client_balances}) {
+          %tr
+            %td= $balance->{loginid}
+            %td= $balance->{currency}
+            %td= $to_monetary_number_format->($balance->{balance})
```

Comment:
@arcofdescent Thanks for suggestion, will improve it this way in my next release